packages/cursorless-engine/src/testCaseRecorder/TestCaseRecorder.ts

```diff
@@ -368,10 +368,20 @@ export class TestCaseRecorder {
         "testCaseSaved",
         "Cursorless test case saved.",
         "View",
+        "Delete",
       ).then(async (action) => {
         if (action === "View") {
           await ide().openTextDocument(outPath);
         }
+        if (action === "Delete") {
+          await fs.unlink(outPath, (err) => {
+            if (err) {
+              console.log("failed to delete ${outPath}: ${err}");
+            } else {
+              console.log("deleted ${outPath}");
```

Comment:
are these meant to be logging literally `${outPath}`? or are there shenanigans with `console` in VSCode I'm missing?

Reply:
oops, this is my typescript naivete. thanks, fixed in #1719